### ISSUE TITLE
TypeError: create_connection() got an unexpected keyword argument 'loop'

### ISSUE THREAD

**Issue by shsmith** [kyuupichan/electrumx#556]
ElectrumX HEAD fails on Python 3.7 with multiple tracebacks.
```
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:root:task crashed: <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 188, in _monitor_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     if await self._should_drop_peer(peer):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 229, in _should_drop_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     **kwargs) as session:
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 314, in __aenter__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.create_connection()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 308, in create_connection
Aug 05 07:05:53 abc1 electrumx_server[17707]:     self_func, self.host, self.port, loop=self.loop, **self.kwargs)
Aug 05 07:05:53 abc1 electrumx_server[17707]: TypeError: create_connection() got an unexpected keyword argument 'loop'
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:TaskGroup:task crashed: <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 188, in _monitor_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     if await self._should_drop_peer(peer):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 229, in _should_drop_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     **kwargs) as session:
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 314, in __aenter__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.create_connection()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 308, in create_connection
Aug 05 07:05:53 abc1 electrumx_server[17707]:     self_func, self.host, self.port, loop=self.loop, **self.kwargs)
Aug 05 07:05:53 abc1 electrumx_server[17707]: TypeError: create_connection() got an unexpected keyword argument 'loop'
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:root:task crashed: <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 188, in _monitor_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     if await self._should_drop_peer(peer):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 229, in _should_drop_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     **kwargs) as session:
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 314, in __aenter__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.create_connection()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 308, in create_connection
Aug 05 07:05:53 abc1 electrumx_server[17707]:     self_func, self.host, self.port, loop=self.loop, **self.kwargs)
Aug 05 07:05:53 abc1 electrumx_server[17707]: TypeError: create_connection() got an unexpected keyword argument 'loop'
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:TaskGroup:task crashed: <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 188, in _monitor_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     if await self._should_drop_peer(peer):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py", line 229, in _should_drop_peer
Aug 05 07:05:53 abc1 electrumx_server[17707]:     **kwargs) as session:
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 314, in __aenter__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.create_connection()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/session.py", line 308, in create_connection
Aug 05 07:05:53 abc1 electrumx_server[17707]:     self_func, self.host, self.port, loop=self.loop, **self.kwargs)
Aug 05 07:05:53 abc1 electrumx_server[17707]: TypeError: create_connection() got an unexpected keyword argument 'loop'
Aug 05 07:05:53 abc1 electrumx_server[17707]: INFO:SessionManager:closing down RPC, TCP, SSL listening servers
Aug 05 07:05:53 abc1 electrumx_server[17707]: INFO:PeerManager:[electroncash.dk:50002 SSL] verified in 1.9s
Aug 05 07:05:53 abc1 electrumx_server[17707]: INFO:PeerManager:[electrum.imaginary.cash:50002 SSL] verified in 1.9s
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:root:task crashed: <Task finished coro=<SessionManager.serve() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/session.py:403> exception=TaskGroupError([<Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>, <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>])>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/session.py", line 432, in serve
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await group.spawn(self._restart_if_paused())
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 263, in __aexit__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.join(wait=self._wait)
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 237, in join
Aug 05 07:05:53 abc1 electrumx_server[17707]:     raise TaskGroupError(bad_tasks)
Aug 05 07:05:53 abc1 electrumx_server[17707]: aiorpcx.curio.TaskGroupError: TaskGroupError(TypeError)
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:TaskGroup:task crashed: <Task finished coro=<SessionManager.serve() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/session.py:403> exception=TaskGroupError([<Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>, <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>])>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/session.py", line 432, in serve
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await group.spawn(self._restart_if_paused())
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 263, in __aexit__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.join(wait=self._wait)
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 237, in join
Aug 05 07:05:53 abc1 electrumx_server[17707]:     raise TaskGroupError(bad_tasks)
Aug 05 07:05:53 abc1 electrumx_server[17707]: aiorpcx.curio.TaskGroupError: TaskGroupError(TypeError)
Aug 05 07:05:53 abc1 electrumx_server[17707]: INFO:BlockProcessor:flushing to DB for a clean shutdown...
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:root:task crashed: <Task finished coro=<Controller.serve() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/controller.py:80> exception=TaskGroupError([<Task finished coro=<SessionManager.serve() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/session.py:403> exception=TaskGroupError([<Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>, <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>])>])>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/controller.py", line 114, in serve
Aug 05 07:05:53 abc1 electrumx_server[17707]:     serve_externally_event.set()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 263, in __aexit__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.join(wait=self._wait)
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 237, in join
Aug 05 07:05:53 abc1 electrumx_server[17707]:     raise TaskGroupError(bad_tasks)
Aug 05 07:05:53 abc1 electrumx_server[17707]: aiorpcx.curio.TaskGroupError: TaskGroupError(TaskGroupError)
Aug 05 07:05:53 abc1 electrumx_server[17707]: ERROR:TaskGroup:task crashed: <Task finished coro=<Controller.serve() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/controller.py:80> exception=TaskGroupError([<Task finished coro=<SessionManager.serve() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/session.py:403> exception=TaskGroupError([<Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>, <Task finished coro=<PeerManager._monitor_peer() done, defined at /usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/peers.py:185> exception=TypeError("create_connection() got an unexpected keyword argument 'loop'")>])>])>
Aug 05 07:05:53 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:05:53 abc1 electrumx_server[17707]:     task.result()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/server/controller.py", line 114, in serve
Aug 05 07:05:53 abc1 electrumx_server[17707]:     serve_externally_event.set()
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 263, in __aexit__
Aug 05 07:05:53 abc1 electrumx_server[17707]:     await self.join(wait=self._wait)
Aug 05 07:05:53 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 237, in join
Aug 05 07:05:53 abc1 electrumx_server[17707]:     raise TaskGroupError(bad_tasks)
Aug 05 07:05:53 abc1 electrumx_server[17707]: aiorpcx.curio.TaskGroupError: TaskGroupError(TaskGroupError)
Aug 05 07:06:03 abc1 electrumx_server[17707]: WARNING:asyncio:<asyncio.sslproto.SSLProtocol object at 0x7f015e6c1780> stalled during handshake
Aug 05 07:06:03 abc1 electrumx_server[17707]: WARNING:asyncio:<asyncio.sslproto.SSLProtocol object at 0x7f014a6b1048> stalled during handshake
Aug 05 07:06:03 abc1 electrumx_server[17707]: WARNING:asyncio:<asyncio.sslproto.SSLProtocol object at 0x7f015eb6abe0> stalled during handshake
Aug 05 07:08:18 abc1 systemd[1]: Stopping Electrumx...
Aug 05 07:08:18 abc1 electrumx_server[17707]: WARNING:Controller:received SIGTERM signal, initiating shutdown
Aug 05 07:08:18 abc1 electrumx_server[17707]: INFO:Controller:shutting down
Aug 05 07:08:18 abc1 electrumx_server[17707]: Traceback (most recent call last):
Aug 05 07:08:18 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/EGG-INFO/scripts/electrumx_server", line 30, in main
Aug 05 07:08:18 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/lib/server_base.py", line 121, in run
Aug 05 07:08:18 abc1 electrumx_server[17707]:     loop.run_until_complete(self._main(loop))
Aug 05 07:08:18 abc1 electrumx_server[17707]:   File "/usr/lib/python3.7/asyncio/base_events.py", line 566, in run_until_complete
Aug 05 07:08:18 abc1 electrumx_server[17707]:     return future.result()
Aug 05 07:08:18 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/electrumX-1.8-py3.7.egg/electrumx/lib/server_base.py", line 104, in _main
Aug 05 07:08:18 abc1 electrumx_server[17707]:     server_task.cancel()
Aug 05 07:08:18 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 263, in __aexit__
Aug 05 07:08:18 abc1 electrumx_server[17707]:   File "/usr/local/lib/python3.7/dist-packages/aiorpcX-0.6.0-py3.7.egg/aiorpcx/curio.py", line 237, in join
Aug 05 07:08:18 abc1 electrumx_server[17707]: aiorpcx.curio.TaskGroupError: TaskGroupError(TaskGroupError)
Aug 05 07:08:18 abc1 electrumx_server[17707]: CRITICAL:root:ElectrumX server terminated abnormally
Aug 05 07:08:18 abc1 systemd[1]: Stopped Electrumx.
```
Tried ElectrumX HEAD on Python 3.6 with aiorpcX HEAD and it does not give this traceback. electrumx_rpc getinfo is not working though.
```
$ electrumx_rpc getinfo
error making request:
```

**Comment by shsmith** [comment]
Found this traceback in the Python 3.6 logs:
```
Aug 05 07:53:18 hsmiths-ubuntu electrumx_server[5643]: INFO:ElectrumX:[211] TCP 123.51.190.194:35430, 14 total
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]: ERROR:root:task crashed: <Task finished coro=<SessionBase._process_messages() done, defined at /usr/local/lib/python3.6/dist-packages/aiorpcX-0.6.0-py3.6.egg/aiorpcx/session.py:83> exception=ProtocolError(-32600, 'response to unsent request: <aiorpcx.jsonrpc.Response object at 0xae439ac4> request ID None')>
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]: Traceback (most recent call last):
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:   File "/usr/local/lib/python3.6/dist-packages/aiorpcX-0.6.0-py3.6.egg/aiorpcx/util.py", line 118, in check_task
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:     task.result()
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:   File "/usr/local/lib/python3.6/dist-packages/aiorpcX-0.6.0-py3.6.egg/aiorpcx/session.py", line 90, in _process_messages
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:     requests = receive_message(item)
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:   File "/usr/local/lib/python3.6/dist-packages/electrumX-1.8-py3.6.egg/electrumx/server/session.py", line 521, in receive_message
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:     return self._receive_message_orig(message)
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:   File "/usr/local/lib/python3.6/dist-packages/aiorpcX-0.6.0-py3.6.egg/aiorpcx/jsonrpc.py", line 721, in receive_message
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:     return self.receive_message(message)
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:   File "/usr/local/lib/python3.6/dist-packages/electrumX-1.8-py3.6.egg/electrumx/server/session.py", line 521, in receive_message
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:     return self._receive_message_orig(message)
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:   File "/usr/local/lib/python3.6/dist-packages/aiorpcX-0.6.0-py3.6.egg/aiorpcx/jsonrpc.py", line 709, in receive_message
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:     return self._receive_response(item, request_id)
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:   File "/usr/local/lib/python3.6/dist-packages/aiorpcX-0.6.0-py3.6.egg/aiorpcx/jsonrpc.py", line 604, in _receive_response
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]:     f'response to unsent request: {response} '
Aug 05 07:53:42 hsmiths-ubuntu electrumx_server[5643]: aiorpcx.jsonrpc.ProtocolError: (-32600, 'response to unsent request: <aiorpcx.jsonrpc.Response object at 0xae439ac4> request ID None')
Aug 05 07:54:02 hsmiths-ubuntu electrumx_server[5643]: INFO:ElectrumX:[212] TCP 123.51.190.194:42374, 12 total
Aug 05 07:54:02 hsmiths-ubuntu electrumx_server[5643]: INFO:ElectrumX:[213] TCP 91.204.150.162:32805, 13 total

```

**Comment by kyuupichan** [comment]
Thanks.  The first is a bug in aiorpcx, and wasn't noticed despite the 100% test coverage because of a flawed test.  I'll fix the test and release 0.6.1.

**Comment by kyuupichan** [comment]
These were both issues in aiorpcx.  I've added tests, released 0.6.2 and updated HEAD to require 0.6.2.
